pom.xml

```diff
@@ -418,7 +418,7 @@
 
 	<build>
 
-		<plugins>
+		<pluginManagement><plugins>
```

Comment:
what affect does this have?

Reply:
It doesn't have an effect and it was inserted by mistake.

Repushing branch w/o this commit